website/react-magma-docs/src/pages/api/toggle-button-group.mdx

```diff
@@ -0,0 +1,229 @@
+---
+pageTitle: Toggle ButtonGroup API
+title: Toggle ButtonGroup
+props:
+  - ToggleButtonGroupProps
+---
+
+<DocsHeading type="api">ToggleButtonGroup</DocsHeading>
+
+The Toggle Button Group is a container for multiple Toggle Buttons and enables controlled behaviors throughout the grouping.
+
+## Basic Usage
+
+The ToggleButtonGroup enables styling and selection states across a group of ToggleButtons.
+
+```typescript
+import React from 'react';
+import { ToggleButton, ToggleButtonGroup } from 'react-magma-dom';
+import { CheckIcon } from 'react-magma-icons';
+
+export function Example() {
+  return (
+    <ToggleButtonGroup>
+      <ToggleButton aria-label="Check icon" icon={<CheckIcon />} />
+      <ToggleButton aria-label="Check icon" icon={<CheckIcon />} />
+      <ToggleButton aria-label="Check icon" icon={<CheckIcon />} />
+      <ToggleButton aria-label="Check icon" icon={<CheckIcon />} />
+    </ToggleButtonGroup>
+  );
+}
+```
+
+## No Space
+
+Removes margins and border radius on each button then applies the border-radius around the container.
+
+```typescript
+import React from 'react';
+import { ToggleButton, ToggleButtonGroup } from 'react-magma-dom';
+import { CheckIcon } from 'react-magma-icons';
+
+export function Example() {
+  return (
+    <ToggleButtonGroup noSpace>
+      <ToggleButton aria-label="Check icon" icon={<CheckIcon />} />
+      <ToggleButton aria-label="Check icon" icon={<CheckIcon />} />
+      <ToggleButton aria-label="Check icon" icon={<CheckIcon />} />
+      <ToggleButton aria-label="Check icon" icon={<CheckIcon />} />
+    </ToggleButtonGroup>
+  );
+}
+```
+
+## Single Select
+
+Allows just one toggled button across the group in behavior like a radio.
+
+```typescript
+import React from 'react';
+import { ToggleButton, ToggleButtonGroup } from 'react-magma-dom';
+import { CheckIcon } from 'react-magma-icons';
+
+export function Example() {
+  return (
+    <ToggleButtonGroup singleSelect>
+      <ToggleButton aria-label="Check icon" icon={<CheckIcon />} />
+      <ToggleButton aria-label="Check icon" icon={<CheckIcon />} />
+      <ToggleButton aria-label="Check icon" icon={<CheckIcon />} />
+      <ToggleButton aria-label="Check icon" icon={<CheckIcon />} />
+    </ToggleButtonGroup>
+  );
+}
+```
+
+## Single Select
+
+Enforces one toggled state across a single select.
```

Comment:
I like the description above a little better though: "Allows just one toggled button across the group in behavior like a radio."